### PR TITLE
Bumps to Dart 2.17 minimum and upgrades melos to 3.1.0

### PR DESCRIPTION
**Pull request by ened**

### CODE REVIEW

**Comment by ened** [comment]
@Taym95 @remonh87 - does this change need a changelog in order to be merged? It is the foundation for other upcoming improvements. Thank you.

**Review by remonh87 — APPROVED** [review]
Looks good to me. maybe someone of hue can have a  look at it as well?

File: pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: reactive_ble_workspace
```
Comment (Taym95) [inline pubspec.yaml:1]:
```suggestion
name: flutter_reactive_ble workspace
```

**Review by Taym95 — CHANGES_REQUESTED** [review]

**Review by Taym95 — APPROVED** [review]

**Comment by ened** [comment]
@Taym95 could you to merge ? I do not have the permission. Thank you

**Pull request merged**